pkg/cmd/marathon/app/test_util.go

```diff
@@ -0,0 +1,17 @@
+package app
```

Comment:
I think we should rename this file from `test_util.go` to `app_test.go` or just put this method into any test file. We shouldl not leak it to a production code since it's only for tests.